### DISCUSSION TITLE
how to change to dark mode

### DISCUSSION THREAD

**Discussion by tkfanswind** [spencerwooo/onedrive-vercel-index#245] [Q&A] [answered]
how to change to dark mode?
thanks

**Comment by spencerwooo** [comment, marked as answer]
color theme is determined by your system's preference:

- for windows: https://blogs.windows.com/windowsexperience/2016/08/08/windows-10-tip-personalize-your-pc-by-enabling-the-dark-theme/
- for macos: https://support.apple.com/en-us/HT208976
